modules/tinymce/src/core/demo/ts/demo/FullDemo.ts

```diff
@@ -167,5 +215,5 @@ export default (): void => {
   };
 
   tinymce.init(settings);
-  tinymce.init(Merger.deepMerge(settings, { inline: true, selector: 'div.tinymce' }));
+  // tinymce.init(Merger.deepMerge(settings, { inline: true, selector: 'div.tinymce' }));
```

Comment:
Inline should still work, I remember removing this editor just to test the console on the right editor

Reply:
Moved these changes to a separate demo. Changes in FullDemo are reverted.